hoot-core/src/main/cpp/hoot/core/conflate/highway/HighwayMatchCreator.cpp

```diff
@@ -77,9 +79,9 @@ class HighwayMatchVisitor : public ElementVisitor
    */
   HighwayMatchVisitor(const ConstOsmMapPtr& map,
     vector<const Match*>& result,boost::shared_ptr<HighwayClassifier> c,
-   boost::shared_ptr<SublineStringMatcher> sublineMatcher, Status matchStatus,
+    boost::shared_ptr<SublineStringMatcher> sublineMatcher, Status matchStatus,
```

Comment:
So there was a global search and replace that took place not long ago where ` shared_ptr<` was replaced with `boost::shared_ptr<` and the leading space was removed.  It was in there so that you didn't get `boost::boost::shared_ptr` but it lost a leading space and kills my OCD.  So I'm fixing it where ever I encounter it.